collections/_members/alonso-lopez.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: person
+name: Alonso López Romero
+Bio: I have been a faculty member in IT Engineer programs, since I joined Hermosillo’s Tech University almost 20 years ago. Mainly teaching and getting involved in academic events and activities, also making bonds up close with local and regional IT organizations. In 2016, I founded an academic IT group, for which I am the main coordinator, to develop various projects and collaborations that will benefit our IT programs. Besides being prototyping and developing solutions, I’ve been helping and advising organizations of my local market. But one of my most cherished milestones has been contributing with Layer5’s Meshery, in which I‘ve been involving with the most welcoming community and the greatest talented people I know. Also I’m a professional cellist for about 20 years, playing cello in my state orchestra, and several ensembles. I had the honor of premiering “Ceremonial” (feb. 2020) by composer T. Hori, and "Fantasía Num.1 para violoncello” (dec. 2018) by composer A. Vega. So I’m also passioned about multimedia, arts, entrepreneurship, creative work, video, film and animation.
```

Comment:
@jesuslerma  We are not having the facility of showing a bio in the current site. So, you will have to remove the `bio` data from this PR.
Though, when you send out a PR for the NG site at that time follow this [template](https://github.com/layer5io/layer5/blob/3f738b30b76e0d618910f279572157d5d82738d7/src/collections/members/_member-profile-template/index.mdx) to make the PR.